lib/rpm_package.py

```diff
@@ -130,3 +130,11 @@ def _load(self):
                     distro_version=self.distro.version)
         except TypeError:
             raise exception.PackageDescriptorError(package=self.name)
+
+    @property
+    def version(self):
+        return self.spec_file.query_tag("version")
+
+    @property
+    def release(self):
+        return self.spec_file.query_tag("release").split('.')[0]
```

Comment:
Same here.

Also, the release field should not be meaningful in the context of an rpm without the whole string. That property was not being used in upgrade_versions.py so either remove it or return the whole Release string.